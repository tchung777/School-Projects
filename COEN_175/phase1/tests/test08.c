int main(void)
{
    signed int x, *p;

    x = 123;
    x = -0xAb123;
    x = 1 != 0;
    p = &x;
    x = !!p;
    x >= '\'';
    x <= 0x1aB9dEf9;
    x > 377;
}
