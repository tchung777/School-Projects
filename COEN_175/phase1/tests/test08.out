keyword int
identifier main
operator (
keyword void
operator )
operator {
keyword signed
keyword int
identifier x
operator ,
operator *
identifier p
operator ;
identifier x
operator =
decimal 123
operator ;
identifier x
operator =
operator -
hexadecimal 0xAb123
operator ;
identifier x
operator =
decimal 1
operator !=
decimal 0
operator ;
identifier p
operator =
operator &
identifier x
operator ;
identifier x
operator =
operator !
operator !
identifier p
operator ;
identifier x
operator >=
character '\''
operator ;
identifier x
operator <=
hexadecimal 0x1aB9dEf9
operator ;
identifier x
operator >
decimal 377
operator ;
operator }
